### PR TITLE
add-to-abl to row view

### PR DESCRIPTION
**Pull request by SidelineCory24**
To-do:
- [x] add "+ to ABL" button to Client Profiles row view
- [x] "+ to ABL" should add the client to the ABL and a toast notification should appear
- [x] "- from ABL" should remove the client to the ABL and a toast notification should appear
- [x] styling holds for different screen sizes

### CODE REVIEW

**Comment by SidelineCory24** [comment]
https://user-images.githubusercontent.com/55964163/167039920-848fea9b-03a0-4407-8daa-7825483840f1.mp4

File: src/sass/_bidderPortfolio.scss [hunk 1]
```diff
@@ -218,6 +218,19 @@
...
+
+      .search-as-client-button {
+        float: right;
+        width: 150px
+      }
+    }
+
+    .add-to-abl-button-row {
+      width: 125px;
+    }
```
Comment (scott062) [inline src/sass/_bidderPortfolio.scss:233]: I think we want the buttons to be the same size in row view (to match card view)

Comment (SidelineCory24) [reply src/sass/_bidderPortfolio.scss]: updates made

**Comment by scott062** [comment]
Add media query for 1260px width and that `justify-content: center`
Screen Shot 2022-05-06 at 2 25 38 PM

**Comment by SidelineCory24** [comment]
> Add media query for 1260px width and that `justify-content: center` Screen Shot 2022-05-06 at 2 25 38 PM
> 
> Sorry the buttons look weird when they collapse

done

**Review by scott062 — APPROVED** [review]

**Pull request merged**